exporter/redis.go

```diff
@@ -52,6 +64,50 @@ func (e *Exporter) connectToRedis() (redis.Conn, error) {
 	return c, err
 }
 
+func (e *Exporter) connectToRedisCluster() (redis.Conn, error) {
+	uri := e.redisAddr
+	if strings.Contains(uri, "://") {
```

Comment:
Since you have already copied this code into `TestClusterKeyValuesAndSizes()`, I propose to move the stage of determining the address into a separate method. In this case, it can also be tested and reused during unit testing, e.g. `getClusterAddr()`:
```go
// redis.go
func (e *Exporter) getClusterAddr() string {
	clusterAddr := e.redisAddr
	if strings.Contains(clusterAddr, "://") {
		...
	} else {
		...
	}

	return clusterAddr
}
```

Reply:
I'd suggest to get rid of the copy in `TestClusterKeyValuesAndSizes()` - we control what uri is being passed to the tests and we shouldn't have to sanitize the input there.